simulation/sub8_montecarlo/montecarlo_tests/montecarlo_pd_controller.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+
+PKG = 'sub8_montecarlo'
+NAME = 'test_controller'
+
+import sys
+import unittest
+import rospy
+import rostest
+from sub8_montecarlo_tools import VerifyController
+
+
+class TestController(unittest.TestCase):
+    def setUp(self, *args):
+        '''TODO:
+            - Assert that wrenches within bounds are close to the unbounded least-squares estimate
+            - Make waiting for node functionality into a test_helpers util
+            - Test random wrenches and assert valid behavior
+        '''
+        self.num_runs = rospy.get_param('runs', 4)
+        self.time_limit = rospy.get_param('run_time', 10)
+
+    def test_pd_controller(self):
```

Comment:
This gets kind of weird, because this is a tool for integration-testing the controller, and it's hard to devise useful unittests for this without a controller that is assumed to work...

Hmm...